src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Validation/Narratives/NarrativeHtmlSanitizerTests.cs

```diff
@@ -61,6 +61,15 @@ public void GivenHtmlWithDivAndText_WhenSanitizingHtml_ThenValidationIsSuccessfu
             Assert.Empty(results);
         }
 
+        [Theory]
+        [InlineData("<div><!-- Comment ended with a dash. This error should be ignored ---><not_valid_tag>This tag should return validation error</not_valid_tag></div>")]
```

Comment:
Instead of making a new test, could you move this scenario to the theory on line 35? That theory's name doesn't fit the scenarios it is running. It is for invalid narratives, not just empty divs. If you could update its name to match what it is testing that would be great.

Reply:
Removed the new test and updated the existing test as suggested in latest commit